src/Microsoft.AspNet.SignalR.Client.JS/jquery.signalR.core.js

```diff
@@ -249,7 +249,12 @@
         }
     };
 
-    _pageWindow.load(function () { _pageLoaded = true; });
+    if (typeof _pageWindow.on == "function") {
+        _pageWindow.on("load", function () { _pageLoaded = true; });
```

Comment:
I take it that `.on` isn't provided by jQuery 1.6.4, but `.load` isn't provided by 3.0.0? Might deserve a comment.